CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.3.1](https://github.com/contentstack/contentstack-utils-javascript/tree/v1.3.1) (2023-12-13)
```

Comment:
In the changelog the version is listed as 1.3.1 following 1.1.0. are we missing any released in between ? @harshithad0703

Reply:
ChangeLog was not updated since past releases, will take this up and update in next release